Src/FluentAssertions/Equivalency/ConversionSelector.cs

```diff
@@ -46,6 +46,11 @@ public void IncludeAll()
         inclusions.Add(new ConversionSelectorRule(_ => true, "Try conversion of all members. "));
     }
 
+    /// <summary>
+    /// Instructs the equivalency comparison to try to convert the value of
+    /// a specific member on the expectation object before running any of the other steps.
+    /// </summary>
+    /// <exception cref="ArgumentNullException"><paramref name="predicate"/> is <c>null</c>.</exception>
```

Comment:
Is the lack of a `<param` doc intentional here (just double checking to be sure since I see parameter docs elsewhere)?

Reply:
Not intentionally left out, I simply copied the summary from `SelfReferenceEquivalencyAssertionOptions.WithAutoConversionFor`.